lua/utils/sub_filter_regexs.lua

```diff
@@ -0,0 +1,3 @@
+return {
```

Comment:
List of sub_filter regex's to test. See https://hypothes-is.slack.com/archives/C1MA4E9B9/p1574742802013200.